src/Reference/ContainsMany.php

```diff
@@ -62,10 +62,12 @@ public function ref($defaults = []): Model
         ]));
 
         // set some hooks for ref_model
-        $m->onHook(['afterSave', 'afterDelete'], function ($model) {
-            $rows = $model->persistence->data[$this->table_alias];
-            $this->owner->save([$this->our_field => $rows ?: null]);
-        });
+        foreach ([Model::HOOK_AFTER_SAVE, Model::HOOK_AFTER_DELETE] as $spot) {
```

Comment:
don't we support `onHook(string[], ...)` format anymore?

Reply:
we do not :)